.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## How the feature works? / How did you fix the issue?
+
+
+## Screenshots, video, GIF if applicable
+
+
+## Check list
+- [ ] – unit-tests written
+- [ ] - e2e-tests written
+- [ ] - documentation updated
+- [ ] – PR name follows the pattern '#1234 – issue name'
```

Comment:
#1234 refers to an actual PR , and the preview is like:
![image](https://github.com/epam/ketcher/assets/27288153/fdc5d0a2-266c-4b60-ab5a-d0411abaf103) 
So maybe we can format it as code:
```suggestion
- [ ] PR name follows the pattern `#1234 – issue name`
```